dali/test/python/test_external_source_parallel_utils.py

```diff
@@ -142,6 +142,7 @@ def check_stop_iteration_resume(pipe, batch_size, layout):
         try:
             while True:
                 (r, ) = pipe.run()
+                r = [r.at(i) for i in range(len(r))]
                 output.append(r)
```

Comment:
I think we should do a copy here, for example:
```suggestion
                output.append(np.copy(r.as_array()))
```